### PR TITLE
[doc] Update how-to-build-runtime.md

### PR DESCRIPTION
**Pull request by hseok-oh**
- Fix make command
- Update build result tree
- Update test run result
- Remove deprecated "LD_LIBRARY_PATH" setting

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

File: docs/howto/how-to-build-runtime.md [hunk 1]
```diff
@@ -70,23 +70,23 @@ Unfortunately, the debug build on the x86_64 architecture currently has an error
 
 ```
 $ export BUILD_TYPE=release
-$ make install
+$ make f Makefile.template install
```
Comment (lemmaa) [inline docs/howto/how-to-build-runtime.md:73]:
```suggestion
$ make -f Makefile.template install
```

File: docs/howto/how-to-build-runtime.md [hunk 2]
```diff
@@ -70,23 +70,23 @@ Unfortunately, the debug build on the x86_64 architecture currently has an error
...
 ```
 
 Or you can simply do something like this:
 
 ```
-$ BUILD_TYPE=release make install
+$ BUILD_TYPE=release make f Makefile.template install
```
Comment (lemmaa) [inline docs/howto/how-to-build-runtime.md:79]:
```suggestion
$ BUILD_TYPE=release make -f Makefile.template install
```

**Comment by lemmaa** [comment]
> - Remove deprecated "LD_LIBRARY_PATH" setting

@hseok-oh , I was not aware of this. What is the change?

File: docs/howto/how-to-build-runtime.md [hunk 3]
```diff
@@ -173,25 +185,23 @@ inception_v3.tflite
 The result of running the inception_v3 model using runtime is as follows. Please consider that this is a test that simply checks execution latency without considering the accuracy of the model.
 
 ```
-$ USE_NNAPI=1 LD_LIBRARY_PATH="./Product/out/lib/:$LD_LIBRARY_PATH" ./Product/out
-/bin/tflite_run ./inception_v3.tflite
```
Comment (hseok-oh) [inline docs/howto/how-to-build-runtime.md:177]: >I was not aware of this. What is the change?

@lemmaa Here is the change

**Review by wateret — APPROVED** [review]
LGTM

**Comment by hseok-oh** [comment]
@lemmaa PTAL

**Pull request merged**